src/Microsoft.AspNet.Mvc.Localization/LocalizedHtmlString.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.Rendering;
+
+namespace Microsoft.AspNet.Mvc.Localization
+{
+    /// <summary>
+    /// A <see cref="HtmlString"/> with localized content.
+    /// </summary>
+    public class LocalizedHtmlString : HtmlString
```

Comment:
Hopefully this point will be moot once we move to the single base `ICanHasBeenEncoded` marker which @rynowak proposed and @sornaks is working on as part of #447.